tests/TestCase/Log/Engine/BaseLogTest.php

```diff
@@ -0,0 +1,121 @@
+<?php
+/**
+ * CakePHP(tm) : Rapid Development Framework (http://cakephp.org)
+ * Copyright (c) Cake Software Foundation, Inc. (http://cakefoundation.org)
+ *
+ * Licensed under The MIT License
+ * Redistributions of files must retain the above copyright notice
+ *
+ * @copyright     Copyright (c) Cake Software Foundation, Inc. (http://cakefoundation.org)
+ * @link          http://cakephp.org CakePHP(tm) Project
+ * @since         t.b.d.
+ * @license       http://www.opensource.org/licenses/mit-license.php MIT License
+ */
+namespace Cake\Test\TestCase\Log\Engine;
+
+use Cake\Log\Engine\BaseLog;
+use Cake\ORM\Entity;
+use Cake\TestSuite\TestCase;
+use Psr\Log\LogLevel;
+
+/**
+ * Class BaseLogImpl
+ * Implementation of abstract class {@see Cake\Log\Engine\BaseLog},
+ * required by test case {@see Cake\Test\TestCase\Log\Engine\BaseLogTest}.
+ */
+class BaseLogImpl extends BaseLog
```

Comment:
Isn't it more important that each of the implementations of BaseLog have the unicode handling behavior? Seems like the tests you have here should be a trait that is included in the test cases for each implementation.